zha/application/platforms/fan/__init__.py

```diff
@@ -375,28 +373,15 @@ def async_update(self, _: Any = None) -> None:
         self.maybe_emit_state_changed_event()
 
 
-IKEA_SPEED_RANGE = (1, 10)  # off is not included
-IKEA_PRESET_MODES_TO_NAME = {
-    1: PRESET_MODE_AUTO,
-    2: "Speed 1",
-    3: "Speed 1.5",
-    4: "Speed 2",
-    5: "Speed 2.5",
-    6: "Speed 3",
-    7: "Speed 3.5",
-    8: "Speed 4",
-    9: "Speed 4.5",
-    10: "Speed 5",
-}
-
-
 @MULTI_MATCH(
     cluster_handler_names="ikea_airpurifier",
     models={"STARKVIND Air purifier", "STARKVIND Air purifier table"},
 )
 class IkeaFan(Fan):
     """Representation of an Ikea fan."""
 
+    _attr_supported_features = FanEntityFeature.SET_SPEED | FanEntityFeature.PRESET_MODE
```

Comment:
not used?

Reply:
I think so. Yes.
Left over from adapting from my pre-zha-split version. Will remove.

Reply:
Done